hrjobcontract/CRM/Hrjobcontract/BAO/HRJobContractRevision.php

```diff
@@ -147,5 +118,54 @@ static function importableFields($contactType = 'HRJobContractRevision',
     }
     return $fields;
   }
-  
+
+  /**
+   * Helper function used in updateEffectiveEndDates() loop.
+   * It updates revision entry with calculated 'effective_end_date'
+   * and 'overrided' values basing on specified date of previously iterated
+   * revision and date of currenly iterated revision.
+   * 
+   * @param int $revisionId
+   * @param string $previousEffectiveDate
+   * @param string $currentEffectiveDate
+   */
+  protected static function _updateRevisionByEffectiveDates($revisionId, $previousEffectiveDate, $currentEffectiveDate) {
```

Comment:
Since PHP allows properties and methods visibility to be controlled with public/protected/private, there's no need to use an underscore prefix for private and protected methods.
This also goes against the conventions of the Drupal code standard: https://www.drupal.org/node/608152#naming

> 8 . Protected or private properties and methods should not use an underscore prefix.